packages/applications-service-api/__tests__/unit/repositories/document.backoffice.repository.test.js

```diff
@@ -41,15 +41,50 @@ describe('document repository', () => {
 		it('calls findMany and count with searchTerm if provided', async () => {
 			await getDocuments({
 				caseReference: caseReference,
-				searchTerm: 'someterm'
+				searchTerm: 'some term'
 			});
 
-			expect(mockFindMany.mock.calls[0][0].where.AND[2].OR[0].description.contains).toEqual(
-				'someterm'
-			);
-			expect(mockCount.mock.calls[0][0].where.AND[2].OR[0].description.contains).toEqual(
-				'someterm'
-			);
+			expect(mockFindMany.mock.calls[0][0].where.AND[2].OR).toEqual([
+				{
+					OR: [
+						{ description: { contains: 'some' } },
+						{ descriptionWelsh: { contains: 'some' } },
+						{ author: { contains: 'some' } },
+						{ authorWelsh: { contains: 'some' } },
```

Comment:
Small nitpick but feel free to ignore: Because its so much longer, maybe worth making this into a variable that can be asserted against both mockFIndMany and mockCount.

Reply:
Good idea, done